taskcluster/fpm_taskgraph/gradle.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Copied from https://github.com/mozilla-mobile/fenix/blob/d918cfdf3841a17545053dca2e7480c648ab4f95/taskcluster/fenix_taskgraph/gradle.py. The only change is that `engine` becomes `flavor`